docs/events.rst

```diff
@@ -0,0 +1,27 @@
+Event hooks in Thebelab
+=======================
+
+When Thebelab is launched (with ``thebelab.bootstrap``), it will emit a series
+of events corresponding to the state of the launch process. You can plug into
+these events to control the behavior on your page.
+
+To do so, use the ``status`` event within Thebelab, like so:
+
+.. code-block:: javascript
+
+       thebelab.on("status", function (evt, data) {
+        console.log("Status changed:", data.status, data.message);
+    });
```

Comment:
Out of curiousity, does this work? I wasn't having luck previously getting events working, and I think it's an issue with the jQuery events implementation (issue #191)

Reply:
I think it worked for me. This is the output in sphinx-thebelab:

![image](https://user-images.githubusercontent.com/1839645/86937235-dc126380-c0f3-11ea-9842-500cc726e5c6.png)


here's where I'm using it:

https://github.com/executablebooks/sphinx-thebelab/blob/master/sphinx_thebelab/_static/thebelab.js#L39